.gitignore

```diff
@@ -9,3 +9,7 @@ __pycache__
 /.cache
 
 /.netlify/build
+
+\.idea/
```

Comment:
don't add this folder to .gitignore if you need this add to .git/info/exclude

Reply:
Done in e90bc94